### ISSUE TITLE
pquery3 is creating empty node0.domain.tld_thread-0.out

### ISSUE THREAD

**Issue by rameshvs02** [Percona-QA/pquery#23]
pquery3 is creating empty node0.domain.tld_thread-0.out inside log directory.

Version
```
[ramesh@qa-hppro1 pwork]$ /home/ramesh/percona-qa/pquery/pquery3-ps -v
* PQuery version: 3.0.0
* PQuery revision: 9bb4917
* PQuery release date: 2018-05-17 18:55:14 +0700
* PQuery MySQL client library: Percona Server v.5.7.21-20
[ramesh@qa-hppro1 pwork]$
```
Config file
```
[master]
# Directory to store logs
logdir = /home/ramesh/pwork/log
# Logfile for master process
logfile = pquery3-master.log
[node0.domain.tld]
database = test
address = localhost
#port = 3306
logdir = /home/ramesh/pwork/log
infile = /home/ramesh/percona-qa/pquery/main.sql
socket = /home/ramesh/pwork/ps57/socket.sock
dbtype = mysql
user = root
password =
threads = 1
queries-per-thread = 10
verbose = No
log-all-queries = Yes
log-failed-queries = Yes
shuffle = Yes
log-query-statistics = No
log-query-duration = No
log-client-output = yes
log-query-number = No
run= Yes
```
Log files
```
[ramesh@qa-hppro1 pwork]$ ls -l log/
total 16
-rw-rw-r--. 1 ramesh ramesh  10 Jul  6 05:30 node0.domain.tld_thread-0.out
-rw-rw-r--. 1 ramesh ramesh 410 Jul  6 05:30 node0.domain.tld_thread-0.sql
-rw-rw-r--. 1 ramesh ramesh 251 Jul  6 05:30 node0.domain.tld_worker.log
-rw-rw-r--. 1 ramesh ramesh 383 Jul  6 05:30 pquery3-master.log
[ramesh@qa-hppro1 pwork]$

[ramesh@qa-hppro1 pwork]$ cat log/node0.domain.tld_thread-0.out










[ramesh@qa-hppro1 pwork]$ cat log/node0.domain.tld_worker.log
-> Successfully connected to Localhost via UNIX socket
-> Server version: 5.7.21-20-debug MySQL Community Server (GPL)
-> Loaded 86314 lines from /home/ramesh/percona-qa/pquery/main.sql
-> WORKER SUMMARY: 7/10 queries failed, (30.00% were successful)
[ramesh@qa-hppro1 pwork]$


[ramesh@qa-hppro1 pwork]$ cat log/pquery3-master.log
* PQuery version: 3.0.0
* PQuery revision: 9bb4917
* PQuery revision date: 2018-05-17 18:55:14 +0700
* PQuery MySQL client library: Percona Server v.5.7.21-20
* Pquery master with PID 22169 started at Fri Jul  6 05:30:23 2018

-> Checking node0.domain.tld params...
-> Running worker for node0.domain.tld
-> Waiting for created worker 22170
=> Exit status of child with PID 22170: 0
[ramesh@qa-hppro1 pwork]$
````

**Comment by abychko** [comment]
hi @rameshvs02,
please attach also node0.domain.tld_thread-0.sql file.
node0.domain.tld_thread-0.out file is created due to `log-client-output = yes`. but seems you have no output in your 10 queries (of 86314 total).

**Comment by rameshvs02** [comment]
@abychko yes you are right, with 10 queries no output generated. I have tried with 100 queries and output generated but there are blank lines in node0.domain.tld_thread-0.out file.
Attached output and SQL files.

[node0.domain.tld_thread-0_sql.txt](https://github.com/Percona-QA/pquery/files/2172610/node0.domain.tld_thread-0_sql.txt)
[node0.domain.tld_thread-0_log.txt](https://github.com/Percona-QA/pquery/files/2172611/node0.domain.tld_thread-0_log.txt)

**Comment by abychko** [comment]
resolved in d69c3e2f3bdf87d58d743610f8be885bfb84aa77
